api/v1/controllers/lenses.js

```diff
@@ -412,21 +410,9 @@ module.exports = {
         });
 
       if (featureToggles.isFeatureEnabled('returnUser')) {
-        authUtils.getUser(req)
-        .then((user) => {
-          if (user) {
-            seqObj.installedBy = user.id;
-          }
-
-          return createLens();
-        })
-        .catch((err) => {
-          if (err.status === httpStatus.FORBIDDEN) {
-            return createLens();
-          }
-
-          return u.handleError(next, err, helper.modelName);
-        });
+        const user = req.user;
+        seqObj.installedBy = user.id;
```

Comment:
what if user is undefined? can the user be undefined when the returnUser toggle is true?

Reply:
The user object is always attached to the req by the verifyUserToken in jwtUtil. So, req.user will never be undefined.